src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -682,13 +682,10 @@ export class ZaakViewComponent
             ),
           );
         }
-        if (this.hasZaakData()) {
-          const title = this.zaak.rechten.wijzigenZaakdata
-            ? "actie.zaakdata.wijzigen"
-            : "actie.zaakdata.bekijken";
+        if (this.hasZaakData() && this.zaak.rechten.wijzigenZaakdata) {
           this.menu.push(
             new ButtonMenuItem(
-              title,
+              "actie.zaakdata.bekijken",
```

Comment:
I forget but I think we now only have the menu option `Zaakdata wijzigen` or not? If so I would suggest to change this label accordingly because it is confusing now?

Reply:
The only option is actually `Zaakdata bekijken`, but you are correct, the label needs to change. Thank you!

Reply:
Done :-)

Reply:
If I log in on TEST with testuser1 I see the menu option `Zaakdata wijzigen`, not `Zaakdata bekijken`?

Reply:
My point was more that if we now only have 1 menu text, it should be in sync with the policy etc. So if the policy is `wijzgingen_zaakdata` I expect this label to have the name `"actie.zaakdata.wijzigen".

Reply:
After this change, only the `beheerder` can view the zaak data, and nobody can change it. So maybe we should do it the other way around and change the policy name?

Reply:
I agree! Otherwise it will be very confusing in the future I think.

Reply:
Done, thanks for the feedback!

Reply:
nice!